tbx/static_src/sass/components/_motif-heading.scss

```diff
@@ -0,0 +1,54 @@
+@use "config" as *;
+
+.motif-heading {
+    $root: &;
+
+    &__drop-cap {
+        color: transparent;
+        background-image: var(--letter-motif-two), var(--letter-motif-one);
+        background-clip: text;
+        background-color: $color--white;
```

Comment:
Could you add a comment here to explain that the background-color is essential as an accessibility fallback for any browsers that don't support background-clip? And should this also be `var(--color--heading)`?

Reply:
Good shout - added in https://github.com/torchbox/torchbox.com/pull/80/commits/f0a0a4ca81dd85da11300360870942aadf162862 and updated to use the css custom property - thanks!